packages/beacon-node/src/util/queue/itemQueue.ts

```diff
@@ -31,7 +31,13 @@ export class JobItemQueue<Args extends any[], R> {
     metrics?: QueueMetrics
   ) {
     this.opts = {...defaultQueueOpts, ...opts};
-    this.opts.signal.addEventListener("abort", this.abortAllJobs, {once: true});
+    this.opts.signal.addEventListener("abort", () => {
+      if(this.opts.dropAllJobOnAbort) {
+        this.dropAllJobs();
```

Comment:
Are we 100% that a `bind` is not necessary here?

Reply:
Yes it's in anonymous function so bind is not needed.